CHANGES.txt

```diff
@@ -1,4 +1,5 @@
 4.0-alpha5
+ * Only calculate dynamicBadnessThreshold once per loop in DynamicEndpointSnitch (CASSANDRA-15798)
```

Comment:
don't we normally do this on commit?

Reply:
Some folks do it ahead of time and some do it during the commit to trunk process (before or after cherry picking/applying the patch). No real standard besides it being included in the final commit being required.